lora/adapter.go

```diff
@@ -158,7 +158,7 @@ func (as *adapterService) ConnectThing(chanID, thingID string) error {
 	}
 
 	if _, err := as.thingsRM.Get(thingID); err != nil {
-		return err
+		return nil
```

Comment:
Why this? If there is an error, return an error, not `nil`.

Reply:
This is to ignore connection events not related to the adapter, otherwise I would log errors for any connection event not related to lora. We could pass Things metadata in connection events to solve this.